src/integration_tests/conftest.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+from unittest.mock import MagicMock
+
+import google.auth.transport.requests
+import google.oauth2.id_token
+import pytest
+import requests
+from fastapi.testclient import TestClient
+from google.cloud import exceptions
+from google.cloud import storage as gcp_storage
+from requests.adapters import HTTPAdapter
+from urllib3 import Retry
+
+storage_client = gcp_storage.Client()
+
+
+def pytest_sessionstart():
+    """Create the buckets before running the test."""
+    if os.environ.get("STORAGE_EMULATOR_HOST"):
+        try:
+            storage_client.create_bucket(os.environ.get("DATASET_BUCKET"))
+        except exceptions.Conflict:
+            pass
+        try:
+            storage_client.create_bucket(os.environ.get("SCHEMA_BUCKET_NAME"))
+        except exceptions.Conflict:
+            pass
+
+
+class RequestWrapper:
+    def __init__(self, api_url, headers=None):
+        self.api_url = api_url
+        self.headers = headers
+        retry_strategy = Retry(
```

Comment:
Just the comment about the rationale about why we do this explaining the delay in the bucket.

Reply:
ok, done